dallinger/command_line.py

```diff
@@ -806,21 +812,22 @@ def export(app, local):
             " --app " + app_name(id),
             shell=True)
 
-        dump_path = dump_database(id)
+    try:
+        subprocess.call([
+            "dropdb",
+            app_name(id),
+        ])
+    except Exception:
+        pass
 
-        try:
-            subprocess.check_call([
-                "pg_restore",
-                "--verbose",
-                "--no-owner",
-                "--clean",
-                "-d",
-                "dallinger",
-                os.path.join("data", id, "data.dump")
-            ])
-
-        except Exception:
-            pass
+    subprocess.call([
```

Comment:
We should use `subprocess.check_call` so that an exception is raised if the command fails, right?